storage/internal/benchmarks/main.go

```diff
@@ -21,12 +21,15 @@ import (
 	"fmt"
 	"io"
 	"log"
-	"math/rand"
+	"net/http"
 	"os"
 	"strconv"
 	"strings"
 	"time"
 
+	epb "github.com/cloudprober/cloudprober/probes/external/proto"
+	"github.com/cloudprober/cloudprober/probes/external/serverutils"
```

Comment:
Fix export ordering

Reply:
What do you mean? Should I ungroup these?

Reply:
Oh I guess it's just weird that these are grouped separately; they can go with the other non-core imports below.